inlong-dataproxy/conf/common.properties

```diff
@@ -28,6 +28,8 @@ proxy.cluster.extTag=default=true
 proxy.cluster.inCharges=admin
 # synchronize interval of meta config (millisecond)
 meta.config.sync.interval.ms=10000
+# whether to startup using the local metadata.json file without connecting to the Manager
+startup.using.local.meta.file.enable=false
```

Comment:
It does not need to be divided into too many levels, it can be similar to other config items, using the naming method of small camel case.

Because in common practice, parameter classification generally means that there are other parameters at the same level, such as the above `proxy.cluster` parameter, the reason for classifying the `cluster` is that it also has sub-level parameters such as `name`, `tag`, and `extTag`, etc.

For example, `startup.localMetaFile.enable` may also be enough in this scenario. It clearly indicates whether to enable local meta files at startup.

Reply:
Previously, parameter names were defined in various styles, but now all new parameters are in all lowercase, dotted, easy-to-understand mode, and the old parameters remain unchanged for the time being